libcontainer/cgroups/systemd/v2.go

```diff
@@ -139,7 +140,11 @@ func (m *unifiedManager) Apply(pid int) error {
 	statusChan := make(chan string, 1)
 	if _, err := dbusConnection.StartTransientUnit(unitName, "replace", properties, statusChan); err == nil {
 		select {
-		case <-statusChan:
+		case s := <-statusChan:
+			if s != "done" {
```

Comment:
Is there a constant for this defined somewhere? If not, can create one and add a link to the API where it says it is expected to return `done` here? It may be helpful if we run into any compatibility issues in the future.

Reply:
Even in systemd sources those are not defined, for example, see https://github.com/systemd/systemd/blob/33b58dfb41a0c0db8110f1fc6dd0fc54e0d9cd4a/src/shared/bus-wait-for-jobs.c#L281 (and above).

It is somewhat described in e.g. https://godoc.org/github.com/coreos/go-systemd/dbus#Conn.StartUnit or https://www.freedesktop.org/wiki/Software/systemd/dbus/

Reply:
@mrunalp 
Besides @kolyshkin 's description, we can also Refer to https://github.com/coreos/go-systemd/blob/5a0db84d3dc459ccdc6ffcc44b1c452bf9f171cb/dbus/methods.go#L78-L101

I have add a constant now.

Reply:
add a link to the API address now. @mrunalp